tools/cldr-apps/src/main/webapp/js/new/cldrSurvey.js

```diff
@@ -12,14 +12,118 @@
  */
 
 const cldrSurvey = (function () {
-  const SURVEY_DEBUG = true;
-
   /*
    * INHERITANCE_MARKER indicates that the value of a candidate item is inherited.
    * Compare INHERITANCE_MARKER in CldrUtility.java.
    */
   const INHERITANCE_MARKER = "↑↑↑";
 
+  const xpathMap = new XpathMap(); // TODO: is it really a singleton?
+
+  let wasBusted = false;
+  let didUnbust = false;
+
+  let loadOnOk = null; // TODO: SurveyMain.java writes scripts that try to reference loadOnOk
+
+  let clickContinue = null; // TODO: SurveyMain.java writes scripts that try to reference clickContinue
+
+  let surveyNextLocaleStamp = 0;
+  let surveyNextLocaleStampId = "";
+
+  // hashtable of items already verified
+  let alreadyVerifyValue = {};
+
+  let showers = {};
+
+  let progressWord = null;
+  let ajaxWord = null;
+  let specialHeader = null; // TODO: supposed to be same as specialHeader in cldrStatus.js, or not?
+
+  let saidDisconnect = false;
+
+  let updateParts = null;
+
+  let cacheKillStamp = null;
+
+  const gPopStatus = {
+    unShow: null,
+    lastShown: null,
+    lastTr: null,
+    popToken: 0,
+  };
+
+  /**
+   * Table mapping CheckCLDR.StatusAction into capabilities
+   * @property statusActionTable
+   */
+  var statusActionTable = {
```

Comment:
good candidate for splitting out as static data

Reply:
yes